### PR TITLE
Updating disk details

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
bud-id: #414
Signed-off-by: GowthamShanmugam <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
@shtripat @r0h4n @nnDarshan please review

**Comment by r0h4n** [comment]
Verified?

**Comment by GowthamShanmugam** [comment]
@r0h4n yes

File: tendrl/node_agent/node_sync/__init__.py [hunk 1]
```diff
@@ -247,10 +247,32 @@ def _run(self):
...
                     )
                 )
+                disks = disk_sync.get_node_disks()
                 try:
-                    NS.etcd_orm.client.delete(
-                        ("nodes/%s/Disks") % NS.node_context.node_id,
-                        recursive=True)
+                    all_disk_id = []
+                    all_disk_id.extend(disks["used_disks_id"])
+                    all_disk_id.extend(disks["free_disks_id"])
```
Comment (r0h4n) [inline tendrl/node_agent/node_sync/__init__.py]: ensure all_disk_id is not None

**Pull request merged**